### PR TITLE
ci: update visual regressions to use @web/dev-server

### PR DESCRIPTION
**Pull request by Westbrook**
## Description
The process to generate visual regression was getting a little less than stable, see: https://app.circleci.com/pipelines/github/adobe/spectrum-web-components/4239/workflows/18ff4246-da97-4e1a-b2aa-35c0c726e194. This updates to the "newer" version of `es-dev-server` called `@web/dev-server` and some newer timing mechanisms in the screenshot collecting code to help ensure the process is stable enough to capture actual regressions.

## Motivation and Context
Stability.

## How Has This Been Tested?
The CI is stable in this branch and `westbrook/spectrum-css` where this solution came about.

## Types of changes
- [x] CI only

## Checklist:
- [x] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -93,7 +93,7 @@
         "@open-wc/demoing-storybook": "^2.4.4",
         "@open-wc/polyfills-loader": "^0.3.3",
         "@open-wc/testing": "^2.5.29",
-        "@spectrum-css/table": "^3.0.0-beta.3",
+        "@spectrum-css/table": "^3.0.0-beta.4",
```
Comment (adixon-adobe) [inline package.json:96]: Should this have updated here?

File: test/visual/stories.js [hunk 2]
```diff
@@ -173,9 +173,6 @@ module.exports = [
     'slider--default',
     'slider--gradient',
     'slider--disabled',
-    'slider--color',
-    'slider--color-with-alpha',
-    'slider--color-disabled',
```
Comment (adixon-adobe) [inline test/visual/stories.js:178]: Were these removals intentional?

**Review by adixon-adobe — COMMENTED** [review]
Spotted a couple of changes that look like they don't belong. Otherwise looks good.

**Comment by Westbrook** [comment]
@adixon-adobe just a little early on both of those two points. Updated to exclude them.

**Review by adixon-adobe — APPROVED** [review]
rockin'!

**Pull request merged**